ui/death_knight/frost/sim.ts

```diff
@@ -48,7 +48,7 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecFrostDeathKnight, {
 	),
 	defaults: {
 		// Default equipped gear.
-		gear: Presets.P1_MASTERFROST_GEAR_PRESET.gear,
+		gear: Presets.P3_MASTERFROST_GEAR_PRESET.gear,
```

Comment:
Should this not still be P1 until P3 is actaally released?

Reply:
You're right actually, my bad!